build.sh

```diff
@@ -852,6 +858,8 @@ case "$COMMAND" in
             cp -R include/Realm include/realm
           fi
           touch include/Realm/RLMPlatform.h
+        else
+          echo "let swiftLanguageVersion = \"$(get_swift_version)\"" > RealmSwift/SwiftVersion.swift
```

Comment:
Why is this in the `else` block? I'm not sure how this logic relates to the `without-core` argument.